apis/config/v1alpha1/projectconfig_types.go

```diff
@@ -97,10 +97,19 @@ type MetricsFeatureGates struct {
 	CreatePrometheusRules bool `json:"createPrometheusRules,omitempty"`
 }
 
+type DatasourcesFeatureGates struct {
+	// CreateDatasources defines whether the operator should install Grafana Datasources
+	// to allow the visualization of metrics on Grafana.
+	CreateDatasources bool `json:"createDatasources,omitempty"`
+}
+
 // ObservabilityFeatureGates configures observability of the operator.
 type ObservabilityFeatureGates struct {
 	// Metrics configures metrics of the operator.
 	Metrics MetricsFeatureGates `json:"metrics,omitempty"`
+
+	// Datasources configures datasources of the operator.
+	Datasources DatasourcesFeatureGates `json:"datasources,omitempty"`
```

Comment:
imho we don't need this setting, as the user can enable/disable the creation of the datasource in each TempoStack instance.

Reply:
Yes, I can see it now. Will remove.